### ISSUE TITLE
Abnormal Loss value  during training

### ISSUE THREAD

**Issue by nothingbut1** [Beckschen/TransUNet#154]
Hi, 
Thanks for this great job. 
I've met some error during training. All the parameters are default. After some iteration, the loss suddenly become large. Then came the NaN value.

![image](https://github.com/user-attachments/assets/45adb035-625a-4de9-9963-5835220396a1)

![image](https://github.com/user-attachments/assets/225ef865-5b67-4129-a77e-2bafd6a34887)

The code is running on Windows with vscode.
I've debugged the code, and found the output value of model is all NaN value, i.e. outputs=model(image_batch). So the CrossEntropyLoss is Nan. I don't know how to fix the problem. Any help is greatly appreciated!
